source/_components/lock.kiwi.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+title: "KIWI Lock"
+description: "Instructions on how to integrate KIWI Smart Lock and Smart Entry."
+date: 2017-08-13 17:25
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Lock
+featured: false
+
+ha_release: 0.63.1
```

Comment:
Please update this to the next major release (`0.66` at the time of writing this comment).